cmd/lk/room.go

```diff
@@ -73,8 +73,8 @@ var (
 							TakesFile: true,
 						},
 						&cli.StringFlag{
-							Name:  "room-configuration",
-							Usage: "`NAME` of the room configuration to associate with the created room",
+							Name:  "room-preset",
```

Comment:
should this a minor version bump? not sure if anybody is using this already that would fail with an upgrade, guess it won't be a silent failure, so probably okay

Reply:
yeah, we'd have to do a minor bump. I don't believe anyone's using this